### PR TITLE
Mike/e2e lode

### PR DESCRIPTION
**Pull request by firefrorefiddle**
Closes # .

### Pre-review checklist

- [ ] All code has been formatted using our config (`make format`)
- [ ] Any new API features or modification of existing behavior is covered as defined in our [test plan](https://github.com/Plutonomicon/cardano-transaction-lib/blob/develop/doc/test-plan.md)
- [ ] The changelog has been updated under the `## Unreleased` header, using the appropriate sub-headings (`### Added`, `### Removed`, `### Fixed`), and the links to the appropriate issues/PRs have been included

### CODE REVIEW

File: src/Contract/Test/E2E/Helpers.purs [hunk 1]
```diff
@@ -216,6 +220,40 @@ geroSign gpassword =
...
+  flip inWalletPage $ \page -> do
+    delaySec 0.1
+    void $ doJQ (inputType "radio") click page
+    void $ doJQ (buttonWithText "Continue") click page
+    delaySec 0.1
+    void $ doJQ (inputType "checkbox") click page
+    void $ doJQ (buttonWithText "Connect") click page
+
+lodeSign :: WalletPassword -> RunningExample -> Aff Unit
+lodeSign gpassword =
```
Comment (firefrorefiddle) [inline src/Contract/Test/E2E/Helpers.purs:251]: This is still TBD

File: src/Contract/Test/E2E/Helpers.purs [hunk 2]
```diff
@@ -216,6 +220,40 @@ geroSign gpassword =
...
+
+flintSign :: WalletPassword -> RunningExample -> Aff Unit
+flintSign gpassword =
+  flip inWalletPage $ \gero -> do
+    void $ doJQ (byId "confirm-swap") click gero
+    typeInto (byId "wallet-password") (unwrap gpassword) gero
+    clickButton "Next" gero
+
+lodeConfirmAccess :: RunningExample -> Aff Unit
+lodeConfirmAccess =
```
Comment (firefrorefiddle) [inline src/Contract/Test/E2E/Helpers.purs:241]: This is still TBD

File: src/Contract/Test/E2E/Helpers.purs [hunk 3]
```diff
@@ -216,6 +220,40 @@ geroSign gpassword =
...
+  flip inWalletPage $ \page -> do
+    delaySec 0.1
+    void $ doJQ (inputType "radio") click page
+    void $ doJQ (buttonWithText "Continue") click page
+    delaySec 0.1
+    void $ doJQ (inputType "checkbox") click page
+    void $ doJQ (buttonWithText "Connect") click page
+
+flintSign :: WalletPassword -> RunningExample -> Aff Unit
+flintSign gpassword =
```
Comment (firefrorefiddle) [inline src/Contract/Test/E2E/Helpers.purs:234]: This is still TBD

File: src/Contract/Test/E2E/Helpers.purs [hunk 4]
```diff
@@ -216,6 +220,40 @@ geroSign gpassword =
     typeInto (byId "wallet-password") (unwrap gpassword) gero
     clickButton "Next" gero
 
+flintConfirmAccess :: RunningExample -> Aff Unit
+flintConfirmAccess =
+  flip inWalletPage $ \page -> do
```
Comment (firefrorefiddle) [inline src/Contract/Test/E2E/Helpers.purs:225]: This is still TBD

**Comment by firefrorefiddle** [comment]
test/ctl-e2e-test.sh is the new entrypoint.

**Review by ngua — COMMENTED** [review]
@firefrorefiddle It looks like the `WalletExts` module never committed. That's imported in several places -- could you commit and push it please?

**Comment by ngua** [comment]
Superseded by #987

**Pull request closed without merging**